### PR TITLE
Upgrade to 7.3.8 GA for multiarch

### PR DESCRIPTION
**Pull request by samding01**
[JDG-4225] Upgrade to 7.3.8 GA for multiarch following branch `dategrid73-dev` and build for multiarch
Signed-off-by Sam Ding shding@redhat.com

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+from: registry.redhat.io/ubi8/ubi:latest
 
 labels:
   - name: "com.redhat.component"
     value: "jboss-datagrid-7-datagrid73-openj9-11-openshift-container"
   - name: "org.jboss.product"
     value: "datagrid"
   - name: "org.jboss.product.version"
-    value: "7.3.7.GA"
+    value: "7.3.8.CR1"
```
Comment (Crumby) [inline image.yaml]: It should be GA for CR artefacts so the final image doesn't contain CR label

Comment (Crumby) [reply image.yaml]: Applies also for the changes below

Comment (samding01) [reply image.yaml]: Updated and contains all changes for x86  (exclude the base images)

**Comment by samding01** [comment]
Can you review and get the code merged?
Thanks,

File: modules/datagrid/configuration/added/clustered-openshift.xml [hunk 2]
```diff
@@ -144,11 +144,17 @@
                     <driver name="h2" module="com.h2database.h2">
                         <xa-datasource-class>org.h2.jdbcx.JdbcDataSource</xa-datasource-class>
                     </driver>
+                    <driver name="mysql" module="com.mysql">
```
Comment (Crumby) [inline modules/datagrid/configuration/added/clustered-openshift.xml]: `mysql` and `postresql` drivers were intentionally removed for s390x as they are not available in RHEL8 images. Having there will probably lead to a startup issues

Comment (samding01) [reply modules/datagrid/configuration/added/clustered-openshift.xml]: removed `mysel` and `postresql` sections, plz review.

**Review by Crumby — APPROVED** [review]

**Review by ryanemerson — APPROVED** [review]
LGTM. I'll wait for @osnipso to approve before merging as multiarch is his area of expertise

**Review by osnipso — APPROVED** [review]
LGTM, I'm merging. Thanks @samding01!

**Pull request merged**